tests/modelbench_tests/test_run_journal.py

```diff
@@ -205,7 +205,9 @@ def test_run_item_output_with_sut(self, journal, sut):
         journal.item_entry("an item", tri)
 
         e = journal.last_entry()
-        assert e["sut"] == tri.sut.uid
+        # we are no longer outputting the sut in item blocks, because it's redundant
+        # now that we only include one sut per run
+        assert "sut" not in e
```

Comment:
Generally I don't think we need tests that cover things that shouldn't be there. Although as mentioned elsewhere, I think this probably should still be there.